script.js

```diff
@@ -0,0 +1,128 @@
+// Task 1. Polyfills
+
+// Task 1.1
+console.log("Task 1.1");
+
+const notes = [
+  {
+    id: 1,
+    title: "Recipe",
+    description: "Ingredients include 2 eggs...",
+    pagesCount: 2,
+    isMarked: false,
+    access: [],
+  },
+  {
+    id: 2,
+    title: "Workouts",
+    description: "3 sets of squats...",
+    pagesCount: 1,
+    isMarked: true,
+    access: [],
+  },
+  {
+    id: 3,
+    title: "Passwords",
+    description: "VISA ...",
+    pagesCount: 6,
+    isMarked: true,
+    access: [],
+  },
+  {
+    id: 4,
+    title: "To Do 2021",
+    description: "1. Learn JS...",
+    pagesCount: 3,
+    isMarked: false,
+    access: [],
+  },
+];
+
+Array.prototype.map = function (callback) {
+  const newArray = [];
+  for (let i = 0; i < this.length; i++) {
+    newArray.push(callback(this[i], i, this));
+  }
+  return newArray;
+};
+
+const notesView = notes.map((note) => {
+  return { id: note.id, title: note.title };
+});
```

Comment:
Hey, we do not use full anonyms function syntax when we have only return statement

Check this out:

```suggestion
const notesView = notes.map((note) => ({ id: note.id, title: note.title }));
```

What is more, we can do the following using properties destructure:

```suggestion
const notesView = notes.map(({id, title}) => ({ id, title }));
```

Looks much better, doesn't it? 😸